package.json

```diff
@@ -21,6 +21,7 @@
     "build:docs": "DOCS_ENV=staging parcel build 'packages/@react-{spectrum,aria,stately}/*/docs/*.mdx' 'packages/react-aria-components/docs/*.mdx' 'packages/@internationalized/*/docs/*.mdx' 'packages/dev/docs/pages/**/*.mdx'",
     "test": "yarn jest",
     "test-strict": "cross-env STRICT_MODE=1 yarn jest",
+    "test-storybook": "test-storybook --url http://localhost:9003  --browsers chromium --no-cache",
```

Comment:
Could this start storybook first (if not running), then run the test runner? Seems like we could, but also I don't see anything in their docs about it.

Reply:
I believe we could do something like https://github.com/adobe/react-spectrum/blob/ba727bdc0c4a57626131e84d9c9b661d0b65b754/scripts/verdaccio.sh#L45-L48 where we wait until storybook outputs "Built in" and then run `test-storybook`.